SCSS/_advantages.scss

```diff
@@ -0,0 +1,57 @@
+.advantages {
+    
+    &--box {
+        font-family: 'Raleway', sans-serif;
```

Comment:
czy na pewno potrzebujemy dodawać to font-family? skoro mamy dodany już do znacznika body